### PR TITLE
Init project

### PR DESCRIPTION
**Pull request by mohammadizanloo55**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/mohammadizanloo55/tg-wakatime-bot.git",
+  "author": "mohammadizanloo55 <mohammadizanloo55@gmail.com>",
+  "license": "MIT",
+  "devDependencies": {
+    "@fullstacksjs/eslint-config": "^6.5.2",
+    "eslint": "^7.32.0",
+    "husky": ">=6",
+    "lint-staged": ">=10",
```
Comment (frontendmonster) [inline package.json:12]: This is not a good idea.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    "prettier": "^2.4.1"
+  },
+  "scripts": {
+    "prepare": "husky install",
+    "clean": "prettier '**/*' --write .  --ignore-unknown"
```
Comment (frontendmonster) [inline package.json:17]: We don't need the prettier script, cuz it's integrated into our eslint config.

File: .lintstagedrc.js [hunk 3]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  "*.*": "prettier '**/*' --write --ignore-unknown",
```
Comment (frontendmonster) [inline .lintstagedrc.js:2]: What extensions do we expect here?

File: .lintstagedrc.js [hunk 4]
```diff
@@ -0,0 +1,4 @@
...
+  "*.js": "./node_modules/.bin/eslint --fix",
```
Comment (frontendmonster) [inline .lintstagedrc.js:3]: We can use just `eslint`

File: .husky/pre-commit [hunk 5]
```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
+npx lint-staged
```
Comment (frontendmonster) [inline .husky/pre-commit:5]: Why?

File: .husky/.gitignore [hunk 6]
```diff
@@ -0,0 +1 @@
+_
```
Comment (frontendmonster) [inline .husky/.gitignore:1]: We don't need this in husky@7

File: .gitignore [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+# Stores VSCode versions used for testing VSCode extensions
+.vscode-test
+
+# yarn v2
+.yarn/cache
+.yarn/unplugged
+.yarn/build-state.yml
+.yarn/install-state.gz
+.pnp.*
```
Comment (frontendmonster) [inline .gitignore:118]: Add newline at the end of files (it's better to configure it with editorconfig)

File: package.json [hunk 8]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "tg-wakatime-bot",
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/mohammadizanloo55/tg-wakatime-bot.git",
+  "author": "mohammadizanloo55 <mohammadizanloo55@gmail.com>",
+  "license": "MIT",
+  "devDependencies": {
+    "@fullstacksjs/eslint-config": "^6.5.2",
```
Comment (frontendmonster) [inline package.json:9]: Please configure the package manager not to use the `^` save prefix.
Also, we use npm in our community. So it's better to keep consistency.

**Review by frontendmonster — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mohammadizanloo55) [reply package.json:12]: All extensions prettier can support
I don't understand the drawbacks of this

File: .lintstagedrc.js (hunk 3 above)
Comment (mohammadizanloo55) [reply .lintstagedrc.js:2]: All extensions prettier can support
I don't understand the drawbacks of this

Comment (frontendmonster) [reply .lintstagedrc.js:2]: OK, we can exclude js/ts files cuz they have their own scrip.

File: .husky/pre-commit (hunk 5 above)
Comment (mohammadizanloo55) [reply .husky/pre-commit:5]: I run the npx mrm@2 lint-staged command twice and I think that's why this problem occurred

File: package.json (hunk 1 above)
Comment (mohammadizanloo55) [reply package.json:12]: Why? What is the problem with this?

File: .lintstagedrc.js (hunk 3 above)
Comment (mohammadizanloo55) [reply .lintstagedrc.js:2]: whats about .json files or .md or ...
It is true that there may not be such files in our project at all, but I think handling them is not bad

Comment (mohammadizanloo55) [reply .lintstagedrc.js:2]: whats about .json files or .md or ...
It is true that there may not be such files in our project at all, but I think handling them is not bad

**Review by frontendmonster — COMMENTED** [review]
Great. :ok_hand:

**Pull request merged**